processing/src/test/java/io/druid/segment/CloserRule.java

```diff
@@ -38,8 +37,8 @@ public CloserRule(boolean throwException)
     this.throwException = throwException;
   }
 
-  private static final Logger log = new Logger(CloserRule.class);
-  private final List<AutoCloseable> autoCloseables = new LinkedList<>();
+  private static final Logger LOG = new Logger(CloserRule.class);
```

Comment:
I'll check, but the prior closer rule may have had some flawed logic in it. If it breaks with the prior version I'll comment here (the breaking code may have been removed by this point). If it doesn't break I'll fork this out to its own PR.